mllib/src/main/scala/org/apache/spark/ml/source/libsvm/LibSVMOptions.scala

```diff
@@ -41,11 +41,15 @@ private[libsvm] class LibSVMOptions(@transient private val parameters: CaseInsen
     case o => throw new IllegalArgumentException(s"Invalid value `$o` for parameter " +
       s"`$VECTOR_TYPE`. Expected types are `sparse` and `dense`.")
   }
+
+  val lineSeparator: Option[String] = parameters.get(LINE_SEPARATOR)
```

Comment:
yea, ok. But I couldn't just imagine an usecase to use more than three characters. It's okay to me that we'll follow committers decisions on this. Thanks!